streampipes-extensions-api/src/main/java/org/apache/streampipes/extensions/api/extractor/IStaticPropertyExtractor.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.streampipes.extensions.api.extractor;
+
+import org.apache.streampipes.model.graph.DataSinkInvocation;
+
+public interface IStaticPropertyExtractor extends IParameterExtractor<DataSinkInvocation> {
+}
```

Comment:
It's a bit clearer from an extension developer's point of view since no generics are used - but we should probably rename it to `IAdapterStaticPropertyExtractor` to be consistent with the other extractor in case the `IStaticPropertyExtractor` is only used for adapters...

Reply:
Another PR is fine for - lets create an issue immediately so we won't forget about it